test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -1429,11 +1429,11 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 				azureDiskProvisioner = "kubernetes.io/azure-disk"
 			}
 
-			if isUsingAzureFileCSIDriver, _ = eng.HasAddon(common.AzureFileCSIDriverAddonName); isUsingAzureFileCSIDriver {
-				azureFileProvisioner = "file.csi.azure.com"
-			} else {
-				azureFileProvisioner = "kubernetes.io/azure-file"
-			}
+			// if isUsingAzureFileCSIDriver, _ = eng.HasAddon(common.AzureFileCSIDriverAddonName); isUsingAzureFileCSIDriver {
```

Comment:
Instead of commenting out, could we instead delete this block?

Reply:
done